streams/testing.json

```diff
@@ -190,226 +190,225 @@
-                                "sha256": "77e23a124857b0cb122dc7358fd4b17873939047c717c1be4bac9cd1f429ffc7"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220522.2.1/x86_64/fedora-coreos-36.20220522.2.1-live-rootfs.x86_64.img",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220522.2.1/x86_64/fedora-coreos-36.20220522.2.1-live-rootfs.x86_64.img.sig",
+                                "sha256": "177c45f4f49d9c2bebb33575b5ec0e113b32af098df486d19e4e4a7107ba2556"
                             }
                         },
                         "raw.xz": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220505.2.0/x86_64/fedora-coreos-36.20220505.2.0-metal.x86_64.raw.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220505.2.0/x86_64/fedora-coreos-36.20220505.2.0-metal.x86_64.raw.xz.sig",
-                                "sha256": "718207e87ecee669ceafe4ab09652502aad6495ce74ff8868ea20ad095da5331",
-                                "uncompressed-sha256": "b36c98e6c3b746dcf5eac5cc716a24fb49d6de327178a6561fe86ae39318faf4"
+                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220522.2.1/x86_64/fedora-coreos-36.20220522.2.1-metal.x86_64.raw.xz",
+                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220522.2.1/x86_64/fedora-coreos-36.20220522.2.1-metal.x86_64.raw.xz.sig",
+                                "sha256": "515dd9f506ec50919022a60e87fadb15de3ae02f4641a0a4995320812366ba0a",
+                                "uncompressed-sha256": "1cfcadccda9c39342cef8464bc044b545b884204c5d3df314adebfbde8888871"
                             }
                         }
                     }
                 },
                 "nutanix": {
-                    "release": "36.20220505.2.0",
+                    "release": "36.20220522.2.1",
                     "formats": {
-                        "qcow2.xz": {
+                        "qcow2": {
                             "disk": {
-                                "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220505.2.0/x86_64/fedora-coreos-36.20220505.2.0-nutanix.x86_64.qcow2.xz",
-                                "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/36.20220505.2.0/x86_64/fedora-coreos-36.20220505.2.0-nutanix.x86_64.qcow2.xz.sig",
-                                "sha256": "ebd951bb93a7199ac2df57cb20d65d89a8db8ffef52bebafaa82f109275c8ff6",
-                                "uncompressed-sha256": "b5cd1000c6cd6213583c8a89209eace83b5255685355ecf3b9d18d54efd6b3cd"
```

Comment:
this unexpected change is expected. We no longer have an `uncompressed-sha256` because the artifact isn't externally compressed. See https://github.com/coreos/coreos-assembler/pull/2848